Tasks/GitHubReleaseV0/operations/Helper.ts

```diff
@@ -238,16 +244,29 @@ export class Helper {
     
     /**
      * Returns tag name associated with the commit.
+     * If tagPattern is specified returns tag name
      * If 0 tag found return undefined
      * else if 1 tag found return tag name
      * else throw error
      * @param githubEndpointToken 
      * @param repositoryName 
      * @param commit_sha 
      */
-    private async _getTagForCommit(githubEndpointToken: string, repositoryName: string, commit_sha: string): Promise<string> {
-        let filteredTag: any = await this.filterTag(githubEndpointToken, repositoryName, commit_sha, this._filterTagsByCommitSha);
+    private async _getTagForCommit(githubEndpointToken: string, repositoryName: string, commit_sha: string, tagPattern: string = null): Promise<string> {
+        let filteredTag: any;
 
+        if (!tagPattern) {
+            filteredTag = await this.filterTag(githubEndpointToken, repositoryName, commit_sha, this._filterTagsByCommitSha);
```

Comment:
just initialize a variable to the required callback. Let the call to filterTag be done outside of this if-else check